build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
     id 'com.jfrog.artifactory'
 }
 
-version = project.findProperty('buildVersion') ?: System.getenv("BUILD_VERSION") ?: '1.0-SNAPSHOT'
+version = project.findProperty('buildVersion') ?: System.getenv("BUILD_VERSION") ?: project.findProperty('version')
```

Comment:
мне кажется, что тут вообще не должно быть этой строчки, т.к. version из gradle.properties приезжает средствами gradle и переопределять на уровне сборке надо -Pversion=...

Reply:
I tried removing the `project.findProperty('version')`, expecting that the default version, `1.0-SNAPSHOT` from `gradle.properties`, would be used. However, since the version is specified to come from `buildVersion` in the build script, it ends up being unspecified if `buildVersion` isn’t set.

Additionally, even when providing the `-Pversion` parameter, the version still remains unspecified with the current setup. This suggests that keeping the `project.findProperty('version')` might be necessary to ensure a default version is used when neither `buildVersion` nor `-Pversion` is provided.

I really appreciate your input and am open to any further suggestions to simplify this setup. Thank you!

Reply:
sorry for my comment in Russian :)

Reply:
@adellinekania a week ago we have discussed the same change: https://github.com/octopusden/octopus-license-gradle-plugin/pull/6#discussion_r1742115862

Why have it been added again?

cc @aryabokon

Reply:
My apologies for the confusion. The intention was to use this configuration on TeamCity to set the version, similar to the build version there. However, it turns out that this is unnecessary, as the version can still be directly obtained from the version property instead. Already update the code.